docs-ref-services/ai-formrecognizer-readme-pre.md

```diff
@@ -1,43 +1,45 @@
 ---
 title: Azure Form Recognizer client library for Java
-keywords: Azure, java, SDK, API, azure-ai-formrecognizer, formrecognizer
+keywords: Azure, , SDK, API, azure-ai-formrecognizer, formrecognizer
```

Comment:
Missing Java in the keywords, should we add it back manually?
@sima-zhu

Reply:
Thanks! Please add them manually. Looking into the issue.